benchalerts/benchalerts/pipeline_steps/github.py

```diff
@@ -31,11 +32,14 @@ class GitHubCheckStep(AlertPipelineStep):
         A GitHubRepoClient instance. Either provide this or ``repo``.
     commit_hash
         The commit hash to update. Default is to use the same one that was analyzed in
-        the GetConbenchZComparisonStep earlier in the pipeline.
+        the ``GetConbenchZComparisonStep`` earlier in the pipeline. If
+        ``GetConbenchZComparisonForRunsStep`` is used because the contender lacks commit
+        metadata, ``commit_hash`` must be provided.
     comparison_step_name
-        The name of the ``GetConbenchZComparisonStep`` that ran earlier in the pipeline.
-        Defaults to "GetConbenchZComparisonStep" (which was the default if no name was
-        given to that step).
+        The name of the ``GetConbenchZComparisonStep`` or
+        ``GetConbenchZComparisonForRunsStep`` that ran earlier in the pipeline.
+        Defaults to "GetConbenchZComparisonStep" (which was the default for that step if
+        no name was given to that step).
```

Comment:
Don't love this default anymore, but I'm not seeing a simple obviously better option